test/boulder-config.json

```diff
@@ -110,7 +110,8 @@
           "expiry": "8760h"
         }
       }
-    }
+    },
+    "maxGoroutines": 0
```

Comment:
Please have a nonzero setting for all of these so the limiting code gets exercised.

Also, the name maxGoroutines is misleading. Can you rename it to maxRPCServerGoroutines?